diff --git a/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js b/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js
index 4241e5ef06d27bf41d919776eab94f1f12800681..f878b0fb08b8fdb9726fb31873d605f55e86d1ae 100644
--- a/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js
+++ b/src/private/webapis/idlecallbacks/specs/NativeIdleCallbacks.js
@@ -12,8 +12,6 @@ import type {TurboModule} from '../../../../../Libraries/TurboModule/RCTExport';
 
 import * as TurboModuleRegistry from '../../../../../Libraries/TurboModule/TurboModuleRegistry';
 
-export opaque type IdleCallbackID = mixed;
-
 export type RequestIdleCallbackOptions = {
   timeout?: number,
 };
@@ -27,8 +25,8 @@ export interface Spec extends TurboModule {
   +requestIdleCallback: (
     callback: (idleDeadline: IdleDeadline) => mixed,
     options?: RequestIdleCallbackOptions,
-  ) => IdleCallbackID;
-  +cancelIdleCallback: (handle: IdleCallbackID) => void;
+  ) => mixed;
+  +cancelIdleCallback: (handle: mixed) => void;
 }
 
 export default (TurboModuleRegistry.getEnforcing<Spec>(
